gameconfig.lua

```diff
@@ -0,0 +1,313 @@
+-- Craft ingredients
+
+
+bm = {}
```

Comment:
`bm` should not be global, if global access is needed then sub table under `basic_materials` should be used.

----

Also including reply to another question here, unrelated to above `bm` variable change request:

>>> While the goal of making 'basic_materials' game agnostic is a good idea. There are still 57 instances of "default:" in the "crafts.lua" file.
>>
>> Craft registration actually does not have to be checked, you just wont get recipes if items are missing. Also https://github.com/mt-mods/home_workshop_modpack/pull/5#issuecomment-1027892317
>
> How are you supposed to use basic_materials in other games if the items are only craftable when using the "default:" mod though?
> I figured the point was to have the items always be craftable so everyone can depend on them.

Depending on exact situation either similar way how you implemented it, through separate mod to handle recipes or leave it for game. For current situation would have probably done similar way.

Basically comment was simply just note that comments _"While the goal of making 'basic_materials' game agnostic is a good idea."_ and _"There are still 57 instances of "default:" in the "crafts.lua" file"_ have nothing to do with each other or with mod itself being usable in any other game, all internal logic is available and all items are available anyway.